wikimedia/wikimedia-wordpress-annual-report-plugin
# Map Carousel Block

Implements Map Section/MapBox Setup [#981](https://github.com/humanmade/Wikimedia/issues/981)

- Creates a map that can control a carousel
- Implements the map-info block pattern

To test:
- Visit https://wikimedia.vipdev.lndo.site/wp-admin/options-general.php?page=map_settings and enter a Mapbox API key
- Create a new Report https://wikimedia.vipdev.lndo.site/wp-admin/post-new.php?post_type=wmf-report and insert the `Map` block
- Add the styles to the map block using the block options menu (eg mapbox://styles/mattwatsonhm/clu09j0hw00tf01p7dpw5hyv7) you will need to design these via your Mapbox account. These can provide different styles for different sections. The example is the grey and white style.
- Click the + button under the map to add a marker
- Add an image, and alter the text of the block pattern
- Drag the marker, or select it and choose the address to alter its position
- Note that two markers next to each other will cluster

Demo: https://cln.sh/0VlnGfswSHPNbTgM9qZL
